<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditFrame</class>
 <widget class="QFrame" name="EditFrame">
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="tabButton">
     <property name="toolTip">
      <string>Tab</string>
     </property>
     <property name="text">
      <string>⇥</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="HistoryLineEdit" name="lineEdit">
     <property name="inputMethodHints">
      <set>Qt::ImhNoAutoUppercase</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="sendButton">
     <property name="toolTip">
      <string>Enter</string>
     </property>
     <property name="text">
      <string>↵</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HistoryLineEdit</class>
   <extends>QLineEdit</extends>
   <header>historylineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
